README.rst

```diff
@@ -4,7 +4,7 @@
         :target: https://travis-ci.com/lettucecfd/lettuce
 
 .. .. image:: https://img.shields.io/pypi/v/lettuce.svg
-        :target: https://pypi.python.org/pypi/lettuce
+:target: https://pypi.python.org/pypi/lettuce
```

Comment:
This looks like an error to me :/
Should I remove the rst-file changes entirely from the PR or is there another issue in this lines?

Reply:
this is an error, yes. You can just remove the line altogether; this comes from cookiecutter. The code is not yet hosted on pypi and even then `lettuce` is surely taken.